prov/shm/src/smr_ep.c

```diff
@@ -1698,16 +1324,25 @@ static int smr_ep_ctrl(struct fid *fid, int command, void *arg)
 			domain = container_of(ep->util_ep.domain,
 					      struct smr_domain,
 					      util_domain.domain_fid);
-			ret = smr_ep_srx_context(domain, ep->rx_size,
-						 &ep->srx);
+			ret = util_ep_srx_context(&domain->util_domain,
+					ep->rx_size, SMR_IOV_LIMIT,
+					SMR_INJECT_SIZE, &ep->srx);
```

Comment:
oops alignment. same on L1334-L1336, and many other places...

Reply:
I usually don't align with the parantheses with functions that are either long or indented a lot to reduce line count